html/captive-portal/lib/captiveportal/PacketFence/Controller/Root.pm

```diff
@@ -55,6 +55,7 @@ captiveportal::PacketFence::Controller::Root - Root Controller for captiveportal
 
 sub auto : Private {
     my ( $self, $c ) = @_;
+    $c->stash->{statsd_timer} = pf::StatsD::Timer->new({ 'stat' => 'captiveportal/' . $c->request->path });
```

Comment:
that would be better if put on level 6 

Otherwise that will create a DB for every URL we have by default and we're not currently displaying those